lib/address/bech32.ts

```diff
@@ -0,0 +1,48 @@
+import { bech32 } from '@scure/base';
+
+import config from 'configs/app';
+import bytesToHex from 'lib/bytesToHex';
+import hexToBytes from 'lib/hexToBytes';
+
+export const DATA_PART_REGEXP = /^[\da-z]{38}$/;
+
+export function toBech32Address(hash: string) {
+  if (config.UI.views.address.hashFormat.bech32Prefix) {
+    try {
+      const words = bech32.toWords(hexToBytes(hash));
+      return bech32.encode(config.UI.views.address.hashFormat.bech32Prefix, words);
+    } catch (error) {}
+  }
+
+  return hash;
+}
+
+export function isBech32Address(hash: string) {
+  if (!config.UI.views.address.hashFormat.bech32Prefix) {
+    return false;
+  }
+
+  if (!hash.startsWith(`${ config.UI.views.address.hashFormat.bech32Prefix }1`)) {
```

Comment:
maybe declare that `${ bech32Prefix }1` string in the config? it's repeated several times and it's not obvious watch does '1' mean